ament_clang_format/ament_clang_format/main.py

```diff
@@ -54,7 +82,10 @@ def main(argv=sys.argv[1:]):
     report = []
 
     # invoke clang_format
-    style = '{BasedOnStyle: Google}'
+    with open(args.config_file, 'r') as h:
+        content = h.read()
+    data = yaml.load(content)
+    style = yaml.dump(data, default_flow_style=True, width=float('inf'))
```

Comment:
Does this work? I was playing with reviving this package a couple of weeks ago, but I was unable to get into a situation where it would take my configuration file, it's really dumb that you can't just give it a file here... So you're dumping the configuration to a string and passing the whole thing as an argument?